tooling/minimize/tests/pass/tree_borrows/sb_fails.rs

```diff
@@ -60,6 +57,20 @@ mod return_invalid_mut {
     }
 }
 
+mod static_memory_modification {
+    // Copied directly from fail/stacked_borrows/static_memory_modification.rs
+    // Version that fails TB: fail/tree_borrows/static_memory_modification.rs
+    static X: usize = 5;
+
+    #[allow(mutable_transmutes)]
+    pub fn main() {
+        let _x = unsafe {
+            std::mem::transmute::<&usize, &mut usize>(&X) // In SB this mutable reborrow fails.
+            // But in TB we are allowed to transmute as long as we don't write.
+        };
```

Comment:
Can you add an `assert!(*x == 5);`? Seems better to also check that this actually does a read.

I will do the same on the Miri side.

Reply:
In fact, make that `*&*x == 5`. Just to add an extra reborrow.

Reply:
Done